mirror/mirror.go

```diff
@@ -26,6 +26,10 @@ const (
 
 var (
 	validID = regexp.MustCompile(`^[a-z0-9_-]+$`)
+	client  = &http.Client{
+		Timeout:   60 * time.Minute,
+		Transport: http.DefaultTransport,
```

Comment:
1)
Why do `Mirror`s share the only one `client`?
I think each `Mirror` should have its own `http.Client`.

2)
And specify why using `http.DefaultTransport` instead of [this](https://github.com/cybozu-go/aptutil/blob/d480efb3d9d46f24a260679b8de2567bf1f8ed09/mirror/mirror.go#L91-L94), please.